src/app/Dashboard/CreateGithubIssue.tsx

```diff
@@ -0,0 +1,97 @@
+import { PageSection, Form, TextInput, FormGroup, Button, ActionGroup, Modal, ModalVariant } from '@patternfly/react-core';
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const CreateGithubIssue: React.FunctionComponent = () => {
+    const [username, setUsername] = useState<string>("");
+    const [repositoryName, setRepositoryName] = useState<string>("");
+    const [title, setTitle] = useState<string>("");
+    const [body, setBody] = useState<string>("");
+    const [token, setToken] = useState<string>("");
+    const [modalIsOpen,setModalIsOpen] = useState(false);
+    const [response, setResponse] = useState([]);
+
+    const onChange = (value) => {
+        setUsername(value);
+    }
+
+    const onChangeRepositoryname = (value) => {
+        setRepositoryName(value);
+
+    }
+
+    const onChangeTitle = (value) => {
+        setTitle(value)
+    }
+
+    const onChangeBody = (value) => {
+        setBody(value);
+    }
+
+    const onChangeToken = (value) => {
+        setToken(value);
+    }
+
+
+
+    const onClick = () => {
+        axios.post(`https://api.github.com/repos/${username}/${repositoryName}/issues`, {title: `${title}`,body: `${body}`}, {headers: {'Authorization': `Bearer ${token}`}}).then(response => {
+            setResponse(response.data);
+            console.log(response.data)
+            setModalIsOpen(true);
+
+        });
+
+    }
+
+    const closeModal = () => {
+        setModalIsOpen(false);
+        setResponse([]);
+      }
+    const getLink =() => {
+        const issueLink = "https://github.com/" +username+ "/" + repositoryName + "/issues/" + response["number"]
```

Comment:
A better way to do this would be template literals.
```suggestion
        const issueLink =  `https://github.com/${username}/${repositoryName}/issues/${response["number"]}` 
```
[Link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Awesome! Thanks!! I will update :)